index.js

```diff
@@ -26,5 +26,6 @@ module.exports = {
     combineMeshes : require('./lib/combineMeshes'),
     combinePrimitives : require('./lib/combinePrimitives'),
     removeUnusedVertices : require('./lib/removeUnusedVertices'),
-    OptimizationStatistics : require('./lib/OptimizationStatistics')
+    OptimizationStatistics : require('./lib/OptimizationStatistics'),
+    gltfPipeline : require('./lib/gltfPipeline')
```

Comment:
Keep in mind that it will have stages to do more than optimization, e.g., add AO.